### PR TITLE
Use Laminas instead of Zend

### PR DESCRIPTION
**Pull request by vkhramtsov**
Changed Zend to Laminas because Zend abandoned.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ To use the graph api id method to fetch the share count you need to set up an ap
  * @var array
  */
 private static $configuration = [
-    'cacheClass' => 'Heise\\Shariff\\ZendCache',
+    'cacheClass' => 'Heise\Shariff\LaminasCache',
```
Comment (liayn) [inline README.md]: I propose changing this to \Heise\Shariff\LaminasCache::class instead of fiddling with strings

Comment (vkhramtsov) [reply README.md]: No problem. Updated

**Comment by vkhramtsov** [comment]
@compeak Any news about this PR?

**Comment by compeak** [comment]
Thank you.

**Comment by vkhramtsov** [comment]
@compeak Would you be so kind to create new release?

**Pull request merged**